client/static/about.html

```diff
@@ -38,7 +38,7 @@ <h3 class="md-title">Porybox</h3>
 
     <p>
       <b>Coding</b><br>
-      not_an_aardvark, YaManicKill, and Raia
+      See the <a href="https://github.com/porybox/porybox/graphs/contributors" target="_blank" rel="noreferrer">Contributors</a> page on GitHub for a list of maintainers who contributed to the project.
```

Comment:
Bit of a word choice issue here. "Maintainers" implies a continual relationship. Could we replace this with "list of people who contributed to this project" please?